Package.swift

```diff
@@ -96,7 +96,7 @@ let package = Package(
   }(),
 
   dependencies: [
-    .package(url: "https://github.com/swiftlang/swift-syntax.git", from: "601.0.0-latest"),
+    .package(url: "https://github.com/swiftlang/swift-syntax.git", from: "602.0.0-latest"),
```

Comment:
Can you update the hardcoded `GIT_TAG` commit hash in `CMakeRules.txt` for TestingMacros target to match this?

Reply:
Will do Monday.